config/global.ini.php

```diff
@@ -385,6 +385,14 @@
 ; This feature will not work with the MYSQLI extension.
 archiving_query_max_execution_time = 7200
 
+
+; Allow you to disable archiving segment on selected plugins
+; When this is on, it won't archive any data for selected plugin if segment is not empty
+; This field accepts more than one plugin separated by comma eg: disable_archiving_segment_for_plugin="Plugin1,Plugin2,Plugin3"
+; This field also accepts array format eg: disable_archiving_segment_for_plugin[]="Plugin1", disable_archiving_segment_for_plugin[]="Plugin2"
+; if you'd like to disable on a specify site, please append the siteId to the [General_siteId]. eg siteId=2 [General_2]
```

Comment:
If we aim to support this feature for all settings in General config, we maybe don't need to mention it in each setting. 🤔

Reply:
@sgiehl do we want to remove this part?

Reply:
Might be better to mention that somewhere else, like in the FAQ

Reply:
@sgiehl right that makes sense, let me convert to FAQ tomorrow. 👍